libs/core/src/lib/nested-list/nested-link/nested-link.directive.ts

```diff
@@ -30,6 +28,13 @@ export class NestedLinkDirective implements OnInit {
     @ContentChild(NestedListTitleDirective)
     title: NestedListTitleDirective;
 
+    /**
+     * @hidden
+     */
+    @HostBinding('attr.tabindex')
+    tabIndex: number = 0;
+
+
     /** Event that is thrown, when any keyboard event is dispatched on this element */
     @Output()
```

Comment:
Whats the value of having this output for the user? If its used only for communication with other components, remove `@Output` decorator.

Reply:
Actually there is not value of having it, this is leftover from old implementation, when link had to communicate with parent compoennts.

Reply:
Same situation as with `keyboardTriggered` event there.